litestar/cli/commands/schema.py

```diff
@@ -31,6 +32,27 @@ def schema_group() -> None:
     """Manage server-side OpenAPI schemas."""
 
 
+def _generate_openapi_schema(app: Litestar, output: Path) -> None:
+    """Generate an OpenAPI Schema."""
+    serializer = get_serializer(app.type_encoders)
+    if output.suffix in (".yml", ".yaml"):
+        content = dump_yaml(
+            msgspec.to_builtins(app.openapi_schema.to_schema(), enc_hook=serializer),
+            default_flow_style=False,
+            encoding="utf-8",
```

Comment:
Not using msgspec's `yaml.encode()` as it passes `sort_keys=False` which is a change in behavior from our previous implementation.